packages/core/src/service/transfer/tonService.ts

```diff
@@ -275,3 +276,141 @@ export const sendTonTransfer = async (
         sendBlockchainMessageRequest: { boc: cell.toString('base64') }
     });
 };
+
+export type TransferMessage = {
```

Comment:
Let's move this functions to new file - "multi Send Service" or similar?

Reply:
Done